### PR TITLE
Card Components

### PR DESCRIPTION
**Pull request by foolhardy21**
This PR has different variants of card component. I am reusing button classes in card.css because button component isn't merged yet. Other than this, feel free to critique. Thanks.

### CODE REVIEW

**Comment by Dev-AkshatGupta** [comment]
Everything looks good Just a small suggestion you are having too much utility classes Try to have them less in count If you can

File: components/card.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    cursor: pointer;
+    color: var(--c-white);
+}
+
+.card-image {
+    width: 100%;
+    object-fit: cover;
+}
+
+.card-3-image {
```
Comment (nayyyhaa) [inline components/card.css]: Can use better naming. as `card-3` is a bit vague as to what this card is. Rest well-written code. Nice work! 👍

**Comment by vrinda-mahajan** [comment]
Just a suggestion, you can provide letter spacing to the card text so that it would look good on the smaller screens also.

**Comment by foolhardy21** [comment]
> Everything looks good Just a small suggestion you are having too much utility classes Try to have them less in count If you can

yeah even i was thinking that. ill make the changes. thanks.

File: components/card.css (hunk 1 above)
Comment (foolhardy21) [reply components/card.css]: ah! i forgot to remove this class as it isn't getting used anymore. thanks for pointing it out.

**Comment by foolhardy21** [comment]
> Just a suggestion, you can provide letter spacing to the card text so that it would look good on the smaller screens also.

sure ill make the changes. thanks

**Pull request merged**